lib/common/theme.dart

```diff
@@ -1,5 +1,6 @@
 import 'package:flutter/foundation.dart';
-import 'package:flutter/material.dart';
+import 'package:provenance_wallet/common/pw_design.dart';
```

Comment:
This imported file references this file - is that bad? In Angular this would be a circular dependency, but it didn't seem to affect my testing and building locally.

Reply:
Probably worth referencing the other files used instead of this file